Inconsistent capitalization info and typos (#154)
mongodb/docs-meta: source/style-guide/style/tables.txt

When creating table titles, use the following guidelines:

- Use headline-style capitalization for table titles. See 

>   :ref:`titles-and-headings` for a list of capitalization guidelines.

Reviewer: [nit, suggestion]

Our [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear) suggests putting the purpose of the link first, followed by the target.

Suggest modifying to:

"For a list of capitalization guidelines, see (link)"

> - Use headline-style capitalization for table titles. See 
>   :ref:`titles-and-headings` for a list of capitalization guidelines.

Reviewer 2: ```suggestion
- Use headline-style capitalization for table titles. To learn more, see 
  :ref:`titles-and-headings`.
```
